cpp/include/cudf/io/orc.hpp

```diff
@@ -0,0 +1,832 @@
+/*
+ * Copyright (c) 2020, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @file orc.hpp
+ * @brief cuDF-IO freeform API
+ */
+
```

Comment:
```suggestion
```

Reply:
Why should this be removed ?

Reply:
non-standard with respect to the rest of cudf, and generally speaking keeping a filename _in_ a file is asking for inconsistency. Why should it remain?

Reply:
will remove it.